thecourseforum/theCourseForum2
# Add social media icons from the home page to the sidebar

## Status
- Done 

## GitHub Issues addressed (`#IssueNumber`)
- This PR closes (#393)

## What I did
- Added the html code for the social media icons from landing.html to sidebar.html and modified it
- Added new css classes to base/sidebar.css for styling

## Screenshots
- Before
<img width="119" alt="Screen Shot 2021-02-28 at 3 04 40 PM" src="https://user-images.githubusercontent.com/25890857/109431793-4a36ce00-79d6-11eb-9189-b2dc08c5f5be.png">
- After
<img width="121" alt="Screen Shot 2021-02-28 at 3 04 16 PM" src="https://user-images.githubusercontent.com/25890857/109431782-3be8b200-79d6-11eb-93a6-30b6dee97b03.png">

## Tests
- Comment out the debug code in sidebar.html
- Click the icons to see if the links go to our social media
- Check to make sure the styling works on different browsers
- Check to see if the styling is mobile compatible

## Questions/Discussions/Notes
- 

## Merging the PR
- Who should merge this PR?
  - The last reviewer to approve can merge it
- Should the commit history be preserved in the base branch, or is it okay to combine all of them into a single commit ("squash-merge")?
  - preserve the history
